src/components/App.js

```diff
@@ -4,7 +4,7 @@ import Display from './Display';
 
 function App() {
   return (
-    <div>
+    <div className="calculator">
```

Comment:
App component needs to use flex display 
![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)

Reply:
done